cmd/config/metrics-profiles/metrics.yml

```diff
@@ -37,6 +37,37 @@
 - query: irate(container_runtime_crio_operations_latency_microseconds{operation_type="network_setup_pod"}[2m]) > 0
   metricName: containerNetworkSetupLatency
 
+# Node metrics: Network
+- query: sum by (instance) ( irate(node_network_receive_bytes_total[5m]) + irate(node_network_transmit_bytes_total[5m]))
+  metricName: nodeNetworkTotal
+
+- query: sum by (instance) ( irate(node_network_receive_bytes_total[5m]))
+  metricName: nodeRxNetwork
+
+- query: sum by (instance) ( irate(node_network_transmit_bytes_total[5m]))
+  metricName: nodeTxNetwork
+
+- query: sum by (instance) ( irate(node_network_drop_total[5m]) )
+  metricName: nodeNetworkDropTotal
+
+- query: sum by (instance) ( irate(node_network_errs_total[5m]) )
+  metricName: nodeNetworkErrTotal
+
+- query: sum by (instance, device) (irate(node_network_receive_bytes_total[5m]) + irate(node_network_transmit_bytes_total[5m]))
```

Comment:
This metric can be obtained with the sum of nodeRxNetworkDevice and nodeTxNetworkDevice